pkg/operator/bootstrap.go

```diff
@@ -31,7 +31,7 @@ func resourceName(namespace string) string {
 	if namespace == "default" {
 		return "docker-registry"
 	}
-	return "image-registry"
+	return "instance"
```

Comment:
i'm surprised not to see this anywhere else... do we not look at the name of the CR when we are handling CR events?

Reply:
nm, looks like we call this function to get the name, from within the controller/eventhandler.